docs/migration-guide.md

```diff
@@ -97,6 +97,11 @@ upgrade that may not be outlined here.
 
 A list of changes is detailed below for migrating:
 
+- **BREAKING CHANGE** The Web, React, Vue, Node.js, and Library presets now use `useBuiltIns: 'usage'` as the
```

Comment:
The breaking changes here are listed are listed in roughly order of potential impact/number of people affected. What do you think about moving the new entry lower down, perhaps below the `With the necessity of external CLIs means the removal of most of the Neutrino CLI's commands` line?

Reply:
ah yep